### ISSUE TITLE
Does it provide the LR scheduler 'ReduceOnPlateau'?

### ISSUE THREAD

**Issue by Jongmin1991** [open-mmlab/mmrotate#807]
### Model/Dataset/Scheduler description

If the loss does not decrease for a certain period in training stage, i want to adjust the learning rate automatically. 

However, there does not seem to be a 'ReuceOnPlateau" in the lr_scheduler list provided by mmcv, mmdetection, mmrotate.

Maybe I couldn't find it ?
If it is available, can you provide an example how it is used ?

Thanks in advance for your answer.

### Open source status

- [ ] The model implementation is available
- [ ] The model weights are available.

**Comment by tikitong** [comment]
@Jongmin1991 by any chance did you manage to get _ReduceOnPlateauLR_ to work on the 'loss'? I have the following error here https://github.com/open-mmlab/mmrotate/issues/1002 Many thanks!
